core/config.cpp

```diff
@@ -0,0 +1,50 @@
+#include "backends/p4tools/modules/p4rtsmith/core/config.h"
+
+#include "lib/error.h"
+
+namespace P4::P4Tools::RTSmith {
+
+void FuzzerConfig::setMaxEntryGenCnt(const int numEntries) {
+    if (numEntries <= 0) {
+        ::P4::error(
+            "P4RuntimeSmith: The maximum number of entries to generate must be a positive "
+            "integer.");
+    }
+    maxEntryGenCnt = numEntries;
+}
+
+void FuzzerConfig::setAttempts(const int numAttempts) {
+    if (numAttempts <= 0) {
+        ::P4::error("P4RuntimeSmith: The number of attempts must be a positive integer.");
+    }
+    attempts = numAttempts;
+}
+
+void FuzzerConfig::setMaxTables(const int numTables) {
+    if (numTables <= 0) {
+        ::P4::error("P4RuntimeSmith: The maximum number of tables must be a positive integer.");
+    }
+    maxTables = numTables;
+}
+
+void FuzzerConfig::setTablesToSkip(const std::vector<std::string> &tables) {
+    tablesToSkip = tables;
+}
+
+void FuzzerConfig::setIsUpdateEntry(const bool updateEntry) { isUpdateEntry = updateEntry; }
+
+void FuzzerConfig::setUpdateCount(const size_t count) {
+    if (count <= 0) {
```

Comment:
I encountered a warning ("Condition is always true") here after changing `count <= 0` to `count < 0`. And I have to obey this since `updateCount` is of type `size_t` in [backends/p4tools/modules/p4rtsmith/targets/bmv2/fuzzer.cpp](https://github.com/fruffy/p4rtsmith/blob/f415759fe70998355016cf9f55c42d90fae4aa80/targets/bmv2/fuzzer.cpp#L23-L24). My initial thought is to accept the `count` as an `int64_t` argument and then `static_cast` it to a `size_t`. But I'm unsure if this is desirable.